docker/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
The Shebang is incorrect for a Docker image based on Debian it should be
```bash
#!/bin/bash
```

Reply:
Fixed by f7afb0deb79e45e42a555ec47abcc9eda5f530bd